controller/metrics/workqueue.go

```diff
@@ -1,101 +0,0 @@
-package metrics
```

Comment:
Not sure that we should just discontinue this set of metrics. It was previously added to inspect controller metrics and some companies are relying on this set already. Aren't we breaking existing clients going in this direction?

Reply:
Metrics comes from https://github.com/kubernetes-sigs/controller-runtime/blob/v0.14.7/pkg/metrics/workqueue.go and have the same name and label set so it will be backward compatible with current metrics